generators/app/index.js

```diff
@@ -45,6 +45,10 @@ module.exports = yeoman.Base.extend({
       if (!this.mavenVersion) {
         throw new Error('We are unable to find a maven executable. A compatible version of maven is required.');
       }
+      try { var configJSON = this.fs.readJSON('.yo-rc.json') } catch (err) { /* ignore */ }
```

Comment:
I copied the configJSON line from subgenerator.js. Eventhough the variable is used below, we are not defining it outside the try block. I wanted to know if this is ok by javascript standards or not